charts/gardener/gardenlet/charts/runtime/templates/deployment.yaml

```
@@ -50,38 +55,19 @@ spec:
 {{ toYaml .Values.global.gardenlet.podAnnotations | indent 8 }}
         {{- end }}
       labels:
-        app: gardener
-        role: gardenlet
-        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-        release: "{{ .Release.Name }}"
-        heritage: "{{ .Release.Service }}"
+{{ include "gardenlet.deployment.labels" . | indent 8 }}
         projected-token-mount.resources.gardener.cloud/skip: "true"
         {{- if .Values.global.gardenlet.podLabels }}
 {{ toYaml .Values.global.gardenlet.podLabels | indent 8 }}
         {{- end }}
     spec:
+{{- include "gardenlet.deployment.topologySpreadConstraints" . | nindent 6 }}
```

For consistency reasons with the rest of the template file:
```suggestion
{{ include "gardenlet.deployment.topologySpreadConstraints" . | indent 6 }}
```